scripts/qa_crowbarsetup.sh

```diff
@@ -4506,13 +4506,25 @@ function onadmin_prepare_crowbar_upgrade
         # move nodes to upgrade mode
         safely crowbar_api_request POST $crowbar_api /installer/upgrade/prepare.json
     else
-        if safely crowbarctl upgrade repocheck crowbar --format plain | grep "missing" ; then
-            complain 11 "Some repository is missing on admin server. Cannot upgrade."
-        fi
-        # FIXME: crowbarctl command can time out easily
-        # Do not use it until https://bugzilla.novell.com/show_bug.cgi?id=997293 is fixed
-        # safely crowbarctl upgrade prepare
-        safely crowbar_api_request POST $crowbar_api /installer/upgrade/prepare.json
+        safely crowbarctl upgrade prepare
+        wait_for 200 5 "grep current_step /var/lib/crowbar/upgrade/progress.yml | grep -v upgrade_prepare" "prepare step to finish"
+    fi
```

Comment:
User can use `crowbarctl upgrade status` to check the status, that is already implemented. 

Also, the command should maybe provide more info when background action is started.

What is still missing, is that crowbarctl should not allow to start another step while some is currently running.